cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/httpframeworkhandler/VertxHttpFrameworkHandler.java

```diff
@@ -0,0 +1,161 @@
+/*
+ * Copyright 2021 LinkedIn Corp. Licensed under the BSD 2-Clause License (the "License"). See License in the project root for license information.
+ */
+
+package com.linkedin.kafka.cruisecontrol.httpframeworkhandler;
+
+import com.google.gson.Gson;
+import com.linkedin.cruisecontrol.httframeworkhandler.CruiseControlHttpSession;
+import com.linkedin.cruisecontrol.httframeworkhandler.HttpFrameworkHandler;
+import com.linkedin.kafka.cruisecontrol.KafkaCruiseControl;
+import com.linkedin.kafka.cruisecontrol.config.KafkaCruiseControlConfig;
+import com.linkedin.kafka.cruisecontrol.servlet.response.ResponseUtils;
+import com.linkedin.kafka.cruisecontrol.vertx.VertxSession;
+import io.vertx.core.MultiMap;
+import io.vertx.ext.web.RoutingContext;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+import static com.linkedin.kafka.cruisecontrol.servlet.UserTaskManager.USER_TASK_HEADER_NAME;
+import static com.linkedin.kafka.cruisecontrol.servlet.response.ResponseUtils.getJsonSchema;
+
+public class VertxHttpFrameworkHandler implements HttpFrameworkHandler<KafkaCruiseControlConfig> {
+
+    static final String[] HEADERS_TO_TRY = {
```

Comment:
This is implemented here and in `ServletHttpFrameworkHandler` too. Please move it to the interface.